pocs/utils/config.py

```diff
@@ -8,7 +8,46 @@
 
 
 def load_config(config_files=None, simulator=None, parse=True, ignore_local=False):
-    """ Load configuation information """
+    """Load configuation information
+
+    This function supports loading of a number of different files. If no options
+    are passed to `config_files` then the default `$POCS/conf_files/pocs.yaml`
+    will be loaded. See Notes for additional information.
+
+    Notes:
+        `config_files` supports a number of options:
```

Comment:
Should that be: `load_config` supports a number of options?

If not, then some other clarification is in order.

Reply:
The note is specifically for all the options that `config_files` supports. I made a small change, does that help anything?